### ISSUE TITLE
Language install fails on Windows when user is on D:

### ISSUE THREAD

**Issue by waynefoltaERI** [explosion/spaCy#1780]
BUG: When I run `python -m spacy download en` it fails with:

```Exception:
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\shutil.py", line 544, in move
    os.rename(src, real_dst)
OSError: [WinError 17] The system cannot move the file to a different disk drive: 'c:\\programdata\\anaconda3\\lib\\site-packages\\thinc' -> 'D:\\Users\\wfolta\\AppData\\Local\\Temp\\pip-4nsiqogi-uninstall\\programdata\\anaconda3\\lib\\site-packages\\thinc'
```

Not clear if you're using `os.move` and it's mistakenly thinking it can `rename` or if you're using `os.rename` and have only tested on systems where the user's directory is in `C:`.

Things are now messed up enough that I can't do the spacy info, below, because thinc is uninstalled.

## Your Environment

Doesn't run due to error hosing things.
* Operating System: Windows
* Python Version Used: 3.6
* spaCy Version Used: 2?
* Environment Information: My home directory is in D: not C:

**Comment by ines** [comment]
Thanks for the report – I think the problem here is similar to #1663 – the problem occurs when pip is trying to install *the model's* dependencies in your conda environment (since the model is a Python package that specifies spaCy as a dependency):

> Under the hood, `spacy download` fetches the download URL of the model file and then installs the package via `pip install`. Since the model package specifies spaCy as a dependency, it looks like pip is actually installing spaCy and its dependency Thinc, which somehow fails in your environment.

> As a solution, you can always [download the model archive files](https://spacy.io/usage/models#download-manual), unzip them and load in the model data manually. You can find the archive files in the [spacy-models repo](https://github.com/explosion/spacy-models), or by clicking the "Release details" button in the [models directory](https://spacy.io/models/en). `spacy.load()` accepts a model package name or a directory, so you can also do:

> ```python
> nlp = spacy.load('/path/to/en_core_web_sm-2.0.0')  # path to data directory contained in archive
> ``` 

> I agree that this is not perfectly convenient and elegant, and we've been thinking about solutions to this problem. There's also an open enhancement issue that discusses better solutions for the model packages in conda environments here: #1686

**Comment by ines** [comment]
Fixed in #1792!
